dist/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>title</title>
+	<link rel="stylesheet" type="text/css" href="dev.css">
+	<link rel="stylesheet" type="text/css" href="index.css">
+</head>
+<body>
+<div id="player"></div>
+<script src="wikiajwplayer.js"></script>
+<script>
+	// this function is callled to inject jwplayer into element with id `player`, second argument is an options object
+	WikiaJWPlayer('player', {
+		// addid this object enables tracking
+		tracking: {
+			// pass track function, data argument structure in tracking.js in track function
+			track: function (data) {
```

Comment:
please provide doc what parameters this funciton get

Reply:
I'd add detailed description in README file. I've created subticket for preparing README file